testing/tests/DevExpress.ui.widgets.editors/autocomplete.markup.tests.js

```diff
@@ -0,0 +1,127 @@
+"use strict";
+
+var $ = require("jquery");
+
+require("ui/autocomplete");
+
+QUnit.testStart(function() {
+    var markup =
+        '<div id="qunit-fixture" class="dx-viewport">\
+            <div id="widget"></div>\
+            <div id="widthRootStyle" style="width: 300px;"></div>\
+            <div id="autocomplete"></div>\
```

Comment:
I think better remove `#autocomplete` and '#autocomplete2' elements. Just replace it with `#widget`